docs/articles/step-4-deploying-to-the-edge.md

```diff
@@ -2,79 +2,109 @@
 title: Step 4 - Deploying to the Edge
 ---
 
-In this guide we'll show you how to deploy your gateway to the edge, at over 200 data-centers around the world. The act of deployment creates new [environments](./environments) and it's worth familiarizing yourself with [how environments work](./environments).
+In this guide we'll show you how to deploy your gateway to the edge, at over 200
+data-centers around the world. The act of deployment creates new
+[environments](./environments) and it's worth familiarizing yourself with
+[how environments work](./environments).
 
-In this tutorial we'll show the default workflow via GitHub, but note that we also support GitLab, BitBucket ([enterprise plan only](https://zuplo.com/pricing)) and [custom CI/CD](./custom-ci-cd).
+In this tutorial we'll show the default workflow via GitHub, but note that we
+also support GitLab, BitBucket
+([enterprise plan only](https://zuplo.com/pricing)) and
+[custom CI/CD](./custom-ci-cd).
 
 To follow this tutorial you'll need
 
-- a github account (it's free, sign up at [github.com](https://github.com)).
-- a zuplo project - complete [Step 1](./step-1-setup-basic-gateway.md), [Step 2](./step-2-add-api-key-auth.md) and [Step 3](./step-3-add-rate-limiting.md) for a great start!
-- to install the [Zuplo GitHub deployer](https://github.com/apps/zuplo/installations/new) to GitHub - you can do that [here](https://github.com/apps/zuplo/installations/new)
+- a GitHub account (it's free, sign up at [github.com](https://github.com)).
+- a zuplo project - complete [Step 1](./step-1-setup-basic-gateway.md),
+  [Step 2](./step-2-add-api-key-auth.md) and
+  [Step 3](./step-3-add-rate-limiting.md) for a great start!
+- to install the
+  [Zuplo GitHub deployer](https://github.com/apps/zuplo/installations/new) to
+  GitHub - you can do that
+  [here](https://github.com/apps/zuplo/installations/new)
 
 ## 1/ Connect to GitHub
 
-First, we'll need to connect your project to GitHub. Go to Settings > Source Control. If your project isn't already connected to GitHub click the **Connect to GitHub** button and follow the auth flow. You'll need to grant permissions for any GitHub organizations you want to work with.
+First, we'll need to connect your project to GitHub. Go to Settings > Source
+Control. If your project isn't already connected to GitHub click the **Connect
+to GitHub** button and follow the auth flow. You'll need to grant permissions
+for any GitHub organizations you want to work with.
 
 ![Connect to Source Control](https://cdn.zuplo.com/assets/54589329-ad98-4fac-9788-74a47a6e5aee.png)
 
 ## 2/ Create the repo and connect the project
 
-Once connected, you'll have the option to connect your project to source control by creating a repo.
+Once connected, you'll have the option to connect your project to source control
+by creating a new repo or connecting to an existing one.
 
 ![Create Repo](https://cdn.zuplo.com/assets/64270bdb-fb05-4718-b1c0-a70baf59629a.png)
```

Comment:
I want to replace this screenshot with the one posted below but I can't get the correct access via jit

![b7b5626e-2d17-48f0-8416-4241fc016055](https://github.com/zuplo/docs/assets/1041842/decd0da9-aa72-489a-8ec1-f70530d8d52c)